libs/domains/organization/src/lib/slices/organization/auth-provider.slice.ts

```diff
@@ -54,15 +54,20 @@ export const authProviderSlice = createSlice({
         state.loadingStatus = 'loading'
       })
       .addCase(fetchAuthProvider.fulfilled, (state: AuthProviderState, action: PayloadAction<GitAuthProvider[]>) => {
-        authProviderAdapter.setAll(state, action.payload)
+        // API return same ids if we have a Github app associated, we need to overide it, because our store get a single id by object (we didn't have the same)
```

Comment:
```suggestion
        // API return same ids if we have a Github app associated, we need to override it, because our store gets a single id by object (we didn't have the same)
```

Reply:
Fixed